docker/ubuntu/Dockerfile

```diff
@@ -43,6 +43,7 @@ FROM ubuntu:18.04
 WORKDIR /app/codechain
 COPY --from=builder /build/codechain/target/release/codechain ./target/release/codechain
 COPY --from=builder /build/codechain/codechain/config/presets/ ./codechain/config/presets
+RUN apt-get update && apt-get install -y libssl-dev
```

Comment:
To utilize Docker cache, move this line before `COPY` lines.

Reply:
I added a new commit.